com.unity.multiplayer.mlapi/Prototyping/NetworkTransform.cs

```diff
@@ -167,13 +167,10 @@ private void UpdateNetVarPerms()
                 default:
                     throw new NotImplementedException($"Authority: {Authority} is not handled");
                 case NetworkAuthority.Server:
-                    m_NetworkState.Settings.WritePermission = NetworkVariablePermission.ServerOnly;
+                    m_NetworkState.Settings.WritePermission = NetworkVariableWritePermission.ServerOnly;
                     break;
                 case NetworkAuthority.Client:
-                    m_NetworkState.Settings.WritePermission = NetworkVariablePermission.OwnerOnly;
-                    break;
-                case NetworkAuthority.Shared:
-                    m_NetworkState.Settings.WritePermission = NetworkVariablePermission.Everyone;
```

Comment:
nitpick: that authority is used in testproject, we might want to remove the balls that use shared authority